src/test/test_ipaddr.cc

```diff
@@ -205,18 +207,13 @@ TEST(CommonIPAddr, TestV4_SkipLoopback)
   result =
     find_ip_in_subnet_list(nullptr, (struct ifaddrs*)&one,
                            CEPH_PICK_ADDRESS_IPV4 | CEPH_PICK_ADDRESS_IPV6,
-                           "10.0.0.0/8", "", true);
+                           "10.0.0.0/8", "");
```

Comment:
We should remove the 10.0.0.0/8 subnet filter here to verify that we correctly prefer the 10.1.2.3 despite the two loopback addresses.